src/user/user.service.ts

```
@@ -0,0 +1,31 @@
+  }
+
+  async findAll() {
+    // return await this.db.select().from(users);
+    return await this.db.query.users.findMany();
+  }
+
+  findOne(id: number) {
+    return `This action returns a #${id} user`;
+  }
+
+  update(id: number, updateUserDto: UpdateUserDto) {
```

ESLint

[error] 'updateUserDto' is defined but never used.

(@typescript-eslint/no-unused-vars)